src/main/java/com/minecolonies/coremod/entity/ai/citizen/cook/EntityAIWorkCook.java

```diff
@@ -136,9 +147,34 @@ protected boolean reachedMaxToKeep()
     @Override
     public void requestSmeltable()
     {
-        if (!getOwnBuilding().hasWorkerOpenRequestsOfType(worker.getCitizenData(), TypeToken.of(getSmeltAbleClass().getClass())))
+        if (!getOwnBuilding().hasWorkerOpenRequestsOfType(worker.getCitizenData(), TypeToken.of(getSmeltAbleClass().getClass()))
+        && !getOwnBuilding().hasWorkerOpenRequestsFiltered(worker.getCitizenData(),
+                req -> req.getShortDisplayString().getSiblings().contains(new TranslationTextComponent(COM_MINECOLONIES_REQUESTS_FOOD))))
         {
-            worker.getCitizenData().createRequestAsync(getSmeltAbleClass());
+            final Map<String, List<ItemStorage>> allowedItems = getOwnBuilding().getCopyOfAllowedItems();
+            if (allowedItems.containsKey(FOOD_LIST))
```

Comment:
We've discussed a bit internally about this, because this could be a food request for over 1000 items.
So what we can do is the following:
a) Add an optional exclusion list to the food request
b) Compare here the size of the total food that exists to the exclusion list size before warning the player about things being turned off.